q2_fragment_insertion/plugin_setup.py

```diff
@@ -48,10 +48,20 @@
 
 plugin.methods.register_function(
     function=q2fi.sepp_16s_greengenes,
-    inputs={'representative_sequences': FeatureData[Sequence]},
+    inputs={'representative_sequences': FeatureData[Sequence],
+            'reference_alignment': FeatureData[AlignedSequence],
+            'reference_phylogeny': Phylogeny[Rooted]},
     parameters=_parameters,
     outputs=_outputs,
-    input_descriptions={'representative_sequences': "The sequences to insert"},
+    input_descriptions={
+        'representative_sequences': "The sequences to insert",
+        'reference_alignment':
+        ('The reference multiple nucleotide alignment used '
+         'to construct the reference phylogeny.'),
+        'reference_phylogeny':
+        ('The rooted reference phylogeny. Must be in sync '
+         'with reference-alignment, i.e. each tip name must'
+         ' have exactly one corresponding row.')},
```

Comment:
row? or record?

Reply:
hm, not sure if I understand the semantic difference between alignment row and record? Do you mean one sequence within the alignment with a "record"?

Reply:
Row is an unusual way to describe a sequence

Reply:
its not really a sequence, since it contains gaps